subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerCachingSmokeTest.groovy

```diff
@@ -104,7 +104,7 @@ class AndroidPluginExpectations {
         ':cityquiz:generateDebugResources': UP_TO_DATE,
         ':cityquiz:javaPreCompileDebug': FROM_CACHE,
         ':cityquiz:mergeDebugAssets': FROM_CACHE,
-        ':cityquiz:mergeDebugJavaResource': FROM_CACHE,
+        ':cityquiz:mergeDebugJavaResource': SUCCESS,
```

Comment:
This is an expected behavior change since this task is now marked as `DoNotCacheByDefault`.
https://issuetracker.google.com/issues/181142260

Caching this task slows down builds.